lib/WalletBackend.ts

```diff
@@ -2502,6 +2502,44 @@ export class WalletBackend extends EventEmitter {
         return this.preparedTransactions.delete(transactionHash);
     }
 
+     /**
+     * Deletes all prepared transactions.
+     *
+     * Example:
+     * ```javascript
+     * wallet.deletePreparedTransactions();
+     * ```
+     *
+     */
+    public deletePreparedTransactions(): void {
+        logger.log(
+            'Function deletePreparedTransactions called',
+            LogLevel.DEBUG,
+            LogCategory.GENERAL,
+        );
+
+        this.preparedTransactions.clear();
+    }
+
+     /**
+     * Gets all prepared transactions.
+     *
+     * Example:
+     * ```javascript
+     * const preparedTransactions = wallet.getPreparedTransactions();
+     * ```
+     *
+     */
+    public getPreparedTransactions(): string[] {
+        logger.log(
+            'Function getPreparedTransactions called',
+            LogLevel.DEBUG,
+            LogCategory.GENERAL,
+        );
+
+        return this.preparedTransactions.values();
```

Comment:
```suggestion
        return Array.from(this.preparedTransactions.values());
```

`.values()` returns an iterator, which we need to convert into an array.